.eslintignore

```diff
@@ -0,0 +1,2 @@
+next.config.js
+postcss.config.js
```

Comment:
Because these are configuration files.
Sometimes a [trailing comma](http://eslint.org/docs/rules/comma-dangle) may cause errors.
I always ignore them.